library/network/src/modules/SuSEFirewall.rb

```diff
@@ -651,7 +651,7 @@ def SaveAndRestartService
     # Create appropriate firewall instance based on factors such as which backends
     # are available and/or running/selected.
     # @return SuSEFirewall2 or SuSEFirewalld instance.
```

Comment:
If we write documentation, people can use our code even without reading its full source, by looking up the API on http://www.rubydoc.info/github/yast/yast-yast2/Yast/SuSEFirewallClass
```rb
# @param backend [Symbol] if not nil, explicitly select :sf2 or :fwd
```